### PR TITLE
Darlp networks link caption settings 1203216026809192 v2

### PR DESCRIPTION
**Pull request by darlp5589**
@triveni-codev @chriscodev 

We created a new branch as fix on the anomaly we experienced on the older #PR510 branch. You may refer to this new branch.

qa1
existing data from server
data on atsc service on edit, then i tried to add linked schedule caption settings, seems that it is not working, it is not adding in the table
[FIXED]

qa2
on newly added atsc channel
try to add valid channel settings witout linked transport
then try to add a new atsc service once added in the table for atsc service
try to link a transportin channel settings, the atsc services table data will be removed
[FIXED]

### CODE REVIEW

**Comment by chriscodev** [comment]
hi @darlp5589 ,

seems the service caption settings is now ok but found a new bug along the process  on checking

from server
	
	i have 2 psip terrestrial transport then i tried to linked them both
	when i tried to update link program and then commit both data that has linked transport seems the link transport will auto unlink when viewing the committed data the second time 


Loom Link: https://www.loom.com/share/e0ba888c5813457cb7fabfa68664e502

**Comment by triveni-codev** [comment]
PR History Tree:
[PR 513](https://github.com/triveni-codev/GBWebApp-dev/pull/513/commits)
↑
[PR 510](https://github.com/triveni-codev/GBWebApp-dev/pull/510/commits) ❌<- Obsolete & completely faced out due to anomaly where 9 files are modified and previous functionalities got removed after updating this one line:
image
↑
[PR 508](https://github.com/triveni-codev/GBWebApp-dev/pull/508/commits)


TLDR: PR 508 is part of PR 513. When PR 513 will be merged, PR 508 codes is already bundled with it.

**Comment by chriscodev** [comment]
hi @darlp5589 , 
LGTM :D

**Review by chriscodev — APPROVED** [review]
LGTM

**Comment by triveni-codev** [comment]
Denoted as approved by main reviewer. Additionally, Final checks and merge problems prevention have been carried out. Merged.
